cli/tui/ongoing/terminal_text.go

```
@@ -0,0 +1,50 @@
+package ongoing
+
+import (
+	"strings"
+	"unicode"
+)
+
+func terminalSafeTextLines(text string) []string {
```

golangci-lint (2.12.2)

[error] func terminalSafeTextLines is unused

(unused)